frontend-v2/src/components/nav.tsx

```diff
@@ -168,13 +221,17 @@ export const Navbar = () => {
             <div className="flex gap-3 justify-between">
               <div className="flex items-center gap-2">
                 <CircleUser className="text-neutral-600" size={20} />
-                {user.Name} ({user.ChapterName})
+                <span>
+                  <span>{user.Name}</span>
+                  {user.role !== 'admin' && <span> ({user.ChapterName})</span>}
```

Comment:
```suggestion
                  {user.role !== 'admin' && <>{' '}<span>({user.ChapterName})</span></>}
```

nit. this makes clear that the space is intentional and necessary, and separates concerns (the need for a space here is not really a concern of the chapter name <span>) which could facilitate any future styling changes.